spec/chewy/type/adapter/active_record_spec.rb

```diff
@@ -86,7 +86,19 @@ def import(*args)
           .to eq([{index: cities.first(2)}, {index: cities.last(1)}])
       end
 
-      specify { expect(import(cities)).to eq([{index: cities}]) }
+      specify do
+        cities
+        expects_db_queries do
+          expect(import(cities, direct_import: false)).to eq([{index: cities}])
+        end
+      end
+      specify do
+        cities
+        expects_no_query do
```

Comment:
Does it really fail if some db queries are made? Doesn't seem so.

Reply:
Changed! Now it does! While linting I removed the raise keyword by mistake